sklearn/tests/test_docstrings.py

```diff
@@ -12,9 +12,7 @@
 numpydoc_validation = pytest.importorskip("numpydoc.validate")
 
 FUNCTION_DOCSTRING_IGNORE_LIST = [
-    "sklearn.covariance._shrunk_covariance.ledoit_wolf",
     "sklearn.covariance._shrunk_covariance.ledoit_wolf_shrinkage",
-    "sklearn.datasets._base.load_linnerud",
```

Comment:
This PR should focus on `ledoit_wolf`. This means we do not need to remove `load_linnerud` from this list.

Reply:
This is indeed the reason it fails.